### DISCUSSION TITLE
💫Stars💫

### DISCUSSION THREAD

**Discussion by LeleDallas** [Schweinepriester/github-profile-achievements#800] [Q&A]
Hello guys!
I wanted to share with you a project I've been working on that I'm really excited about. 
It's a digital version of the popular game `Cards Against Humanity`.

You can find the GitHub repository for the game here: [Cards Against Humanity Game](https://github.com/LeleDallas/Cards-Against-Humanity)
If you have a moment, I'd be thrilled if you could take a look at the repository and leave a star ⭐️.

Thank you so much for your time, and I hope you have a blast playing the game!

**Comment by Schweinepriester** [comment]
See https://github.com/Schweinepriester/github-profile-achievements/discussions/722
